geomstats/geometry/special_euclidean.py

```diff
@@ -46,10 +46,11 @@ class _SpecialEuclideanMatrices(GeneralLinear, LieGroup):
 
     def __init__(self, n):
         super(_SpecialEuclideanMatrices, self).__init__(
-            dim=int((n * (n - 1)) / 2), default_point_type='matrix', n=n + 1)
+            dim=int((n * (n + 1)) / 2), default_point_type='matrix', n=n + 1)
         self.rotations = SpecialOrthogonal(n=n)
         self.translations = Euclidean(dim=n)
         self.n = n
+        self.dim = int((n * (n + 1)) / 2)
```

Comment:
This attribute should have been set while calling super()?

Reply:
For some reason using super does not overwrite the dimension of the parent class GeneralLinear so this results in `SpecialEuclidean(n).dim = n^2` (:fearful: )

Reply:
Hmmm, it'd be great if we can unterstand this, as we might have that problem for other classes too!